web-components/src/components/table/ActionsTable.tsx

```diff
@@ -80,6 +80,7 @@ const ActionsTable: React.FC<React.PropsWithChildren<ActionsTableProps>> = ({
   onTableChange,
   initialPaginationParams = DEFAULT_TABLE_PAGINATION_PARAMS,
   isRoutePagination = false,
+  sortCallbacks = [],
```

Comment:
Not exactly, it's an array of callbacks sorted by column orders.
Since we're iterating over columns to display the header we're relying on this order too for callbacks.
I could turn that into a map or object but it would really be the same thing since we would use index for keys.
Let me know if you still prefer a Map, I don't mind changing it.